### ISSUE TITLE
Wordpress not in sync?

### ISSUE THREAD

**Issue by epicfaace** [libreform/wp-libre-form#164]
It seems like the latest version on wordpress codex (1.4.3) is not in sync with the latest version in github (1.5.0.2). Can you more regularly sync it up?

**Comment by k1sul1** [comment]
We've decided not to upload 1.5 to w.org at this time, because it has/had potential to break existing forms, see recent issues for reference. 

We'll upload 2.0 to w.org, but we'll need some time to complete that first. Breaking changes will ensue, but _some_ of those changes won't affect w.org users as they haven't even seen the features that will change.

_Usually_ those who care about the newest version use Packagist, but I understand that isn't always an option. I released 1.5.0.2 a few days ago in order to include some bugfix commits, but our Packagist integration seems to have broken, as the new release hasn't synced to Packagist, so the only way to get the latest release right now is: https://github.com/libreform/wp-libre-form/releases

I suppose we should rename this package to libreform/wp-libre-form and resubmit this to Packagist under the libreform organization (which I already created) so we could have it synchronized with Packagist once again. @anttiviljami isn't very active any more, and we can't control anything under his Packagist account. 

@timiwahalahti @anttiviljami You ok with "transferring" this to libreform packagist.org organization?
